ground/src/main/res/values/colors.xml

```diff
@@ -23,6 +23,7 @@
   <color name="textOverMap">#FCFDF7</color>
   <color name="blackMapOverlay">#000000</color>
   <color name="lightBackground">#EDEEE9</color>
+  <color name="material_dynamic_neutral50">#747679</color>
```

Comment:
Do we have a dark mode variant for this color? If so you should add it to the night colors file

Reply:
We spoke. We should be using "on-surface" color with a 0.12 opacity for the fill. Thanks!